pegasus/sites.v3/code.org/views/unsupported_browser.haml

```diff
@@ -5,7 +5,15 @@
 
 :javascript
   $(document).ready(function () {
+    if (!window.isUnsupportedBrowser) {
```

Comment:
This is a really small nit, but I would have written:
```javascript
setTimeout(() => {
  if (isUnsupportedBrowser()) {
    $('#warning-banner').show();
  }
}, window.isUnsupportedBrowser ? 0 : 5000);
```
...which doesn't have quite the same meaning (because it introduces a `setTimeout(_, 0)`) but in this case might be good enough, and might be easier to reason about (e.g. we will _always_ run this callback, it's just a question of when).